Gems/MiniAudio/Code/CMakeLists.txt

```
@@ -42,6 +42,8 @@ ly_add_target(
     FILES_CMAKE
         miniaudio_private_files.cmake
         ${pal_dir}/miniaudio_private_files.cmake
+        PLATFORM_INCLUDE_FILES
```

Can you move the idention here?
```suggestion
    PLATFORM_INCLUDE_FILES
```